src/components/ButtonSelectGroup/OptionButton.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import uuidv4 from 'uuid/v4'
-import { Button } from '../../Button'
+import { Button } from '../Button'
```

Comment:
Note: Having things like `Inputs/ButtonSelectGroup` confused react-styleguidist in terms of it's ability to alphabetize the navigation. With more effort, we could override some modules in react-styleguidist, but I've elected to just push up things like `Inputs/ButtonSelectGroup`  to top level: `ButtonSelectGroup` and so much of this PR is just accommodating for these changes in paths.